backend/tracim_backend/command/caldav.py

```diff
@@ -92,6 +92,10 @@ def take_app_action(
                 nb_error_agenda_access += 1
                 print('Cannot access to agenda server: connection error.')
                 logger.exception(self, exc)
+            except Exception as exc:
```

Comment:
@inkhey all the intelligence of this operation must be in an internal module, not in a command. I should be able to create all agendas out of a CLI executed command.

Reply:
it should be in AgendaApi class

Reply:
@lebouquetin how we deal that with proper cli response in terminal ?

Reply:
if you means `print` instead of `logging.info`, I think you can add a parameter to the constructor or to the method call which allow to output to stdout instead of logging.

If you mean something else, let's explain.

Reply:
@lebouquetin Currently log and "normal printed" result are completely different things. Logging in stdout for command can be enabled with debug mode "-d", 
Printed result are like some kind of "frontend" whereas logging is for debugging/admin purpose.
I do not think mix them is a good idea.

Reply:
@buxx Have you got an idea about this ?

Reply:
@inkhey what is the problem when you write "how we deal that with proper cli response in terminal ?" ? You do not expose the problem so it's difficult to find a solution.

Reply:
@lebouquetin I talk about this with @buxx , i will propably do this with another specific log for thing that mean to be printed for console or other thing .

Reply:
will be fixed later, no so trivial : https://github.com/tracim/tracim/issues/1591